### PR TITLE
[INLONG-825][Doc] Add the quick start guide for collecting MySQL and writing to ClickHouse

### LINKED ISSUE #825: [Doc][Translate] Add the quick start guide for collecting MySQL and writing to ClickHouse
### Description

There is a [Chinese document](https://mp.weixin.qq.com/s/gRVY00PyORbKXDo15nvgWQ). Contributors who want to work on this PR could refer to it.

### Are you willing to submit PR?

- [ ] Yes, I am willing to submit a PR!

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://www.apache.org/foundation/policies/conduct)

### PR DESCRIPTION
**Pull request by Amilelne**
Add the quick start guide for collecting MySQL and writing to ClickHouse

fix #825

### CODE REVIEW

File: docs/quick_start/clickHouse_example.md [hunk 1]
```diff
@@ -0,0 +1,96 @@
+---
+title: ClickHouse Example
+sidebar_position: 2
+---
+
+Here we use an example to introduce how to use Apache InLong creating MySQL -> Kafka -> ClickHouse data access.
```
Comment (dockerzhang) [inline docs/quick_start/clickHouse_example.md:6]: MySQL -> Kafka -> ClickHouse data access
->
`MySQL -> Kafka -> ClickHouse` data ingestion

File: docs/quick_start/clickHouse_example.md [hunk 2]
```diff
@@ -0,0 +1,96 @@
...
+![clickhouse](img/clickhouse/data_page.png)
+
+then enter ClickHouse container, check data in table
+![clickhouse](img/clickhouse/data_table.png)
+
+## Questions
+### Task Configuration Failed
+Generally, the MQ or Flink group configuration is incorrect. You can view the error information on the page, or enter the Manager container to view detailed logs.
+
+
```
Comment (dockerzhang) [inline docs/quick_start/clickHouse_example.md:92]: too many empty lines.

File: docs/quick_start/clickHouse_example.md [hunk 3]
```diff
@@ -0,0 +1,96 @@
...
+:::caution
+Since each component reports the ClusterTags as `default_cluster` by default, do not use other names.
+:::
+
+### Register Kafka Cluster
+Click [Clusters]->[Create] on the page to register Kafka Cluster:
+![Create kafka cluster](img/clickhouse/kafka_cluster.png)
+
+### Register Clickhouse DataNodes
+Click [DataNodes]->[Create] on the page to register ClickHouse DataNodes.
```
Comment (dockerzhang) [inline docs/quick_start/clickHouse_example.md:48]: end with ":" ?

File: docs/quick_start/clickHouse_example.md [hunk 4]
```diff
@@ -0,0 +1,96 @@
...
+![Create kafka cluster](img/clickhouse/datanode.png)
+
+## Create Task
+### Create Data Streams Group
+Click [Ingestion]->[Create], inpit ID and select Kafka MQ Type
```
Comment (dockerzhang) [inline docs/quick_start/clickHouse_example.md:53]: ditto

File: docs/quick_start/clickHouse_example.md [hunk 5]
```diff
@@ -0,0 +1,96 @@
+---
+title: ClickHouse Example
```
Comment (dockerzhang) [inline docs/quick_start/clickHouse_example.md:2]: MySQL to ClickHouse

File: i18n/zh-CN/docusaurus-plugin-content-docs/current/quick_start/clickHouse_example.md [hunk 6]
```diff
@@ -0,0 +1,93 @@
+---
+title: 入库 ClickHouse 示例
```
Comment (dockerzhang) [inline i18n/zh-CN/docusaurus-plugin-content-docs/current/quick_start/clickHouse_example.md:2]: MySQL -> Kafka -> ClickHouse 示例

File: docs/quick_start/clickHouse_example.md [hunk 7]
```diff
@@ -0,0 +1,90 @@
+---
+title: MySQL to ClickHouse
```
Comment (dockerzhang) [inline docs/quick_start/clickHouse_example.md:2]: MySQL -> Kafka -> ClickHouse Example

File: i18n/zh-CN/docusaurus-plugin-content-docs/current/quick_start/clickHouse_example.md [hunk 8]
```diff
@@ -0,0 +1,93 @@
...
+sidebar_position: 2
+---
+
+在下面的内容中，我们将通过一个完整的示例介绍如何使用 Apache InLong 创建 MySQL -> Kafka -> ClickHouse 数据链路。
+
+## 环境部署
+### 安装 InLong
+
```
Comment (dockerzhang) [inline i18n/zh-CN/docusaurus-plugin-content-docs/current/quick_start/clickHouse_example.md:10]: Remove this empty line.

File: i18n/zh-CN/docusaurus-plugin-content-docs/current/quick_start/clickHouse_example.md [hunk 9]
```diff
@@ -0,0 +1,93 @@
...
+# 安装 Kafka 
+docker run -d --name kafka -p 9092:9092 -e KAFKA_BROKER_ID=0 -e KAFKA_ZOOKEEPER_CONNECT=zookeeper:2181 --link zookeeper -e KAFKA_ADVERTISED_LISTENERS=PLAINTEXT://kafka:9092 -e KAFKA_LISTENERS=PLAINTEXT://0.0.0.0:9092 -t wurstmeister/kafka
+```
+
+## 集群初始化
+容器启动成功后，访问 InLong Dashboard 地址 http://localhost，并使用以下默认账号登录：
+```
+User: admin
+Password: inlong
+```
```
Comment (dockerzhang) [inline i18n/zh-CN/docusaurus-plugin-content-docs/current/quick_start/clickHouse_example.md:33]: lack one empty line before `###`.

File: i18n/zh-CN/docusaurus-plugin-content-docs/current/quick_start/clickHouse_example.md [hunk 10]
```diff
@@ -0,0 +1,93 @@
...
+### 新建数据流组
+页面点击【数据集成】→【新建数据流组】，填写 ID，选择 MQ 类型为 Kafka 即可：
+![Create datastream](img/clickhouse/create_ingestion.png)
+
+### 新建数据流
+点击【下一步】→【新建数据流】，配置数据流 ID 及测试源字段信息：
+![Create datastream](img/clickhouse/data_stream_config.png)
+
+### 新建数据源
+点击 【新建数据源】→【MySQL】，配置数据源名称、地址、用户密码以及库表信息等
```
Comment (dockerzhang) [inline i18n/zh-CN/docusaurus-plugin-content-docs/current/quick_start/clickHouse_example.md:58]: Add ":" at the end.

File: i18n/zh-CN/docusaurus-plugin-content-docs/current/quick_start/clickHouse_example.md [hunk 11]
```diff
@@ -0,0 +1,93 @@
...
+![Create datastream](img/clickhouse/create_data_source.png)
+
+### 新建数据目标
+点击 【新建数据目标】→【ClickHouse】，配置名称、库表、已创建的 ck 数据节点以及 Schema 映射信息等，完成后点击 【提交审批】
```
Comment (dockerzhang) [inline i18n/zh-CN/docusaurus-plugin-content-docs/current/quick_start/clickHouse_example.md:62]: ditto

File: i18n/zh-CN/docusaurus-plugin-content-docs/current/quick_start/clickHouse_example.md [hunk 12]
```diff
@@ -0,0 +1,93 @@
...
+
+![Create data object](img/clickhouse/create_sink.png)
+
+### 审批数据流
+页面点击【审批管理】->【我的审批】->【详情】->【选择集群标签】->【通过】
```
Comment (dockerzhang) [inline i18n/zh-CN/docusaurus-plugin-content-docs/current/quick_start/clickHouse_example.md:67]: ditto

File: i18n/zh-CN/docusaurus-plugin-content-docs/current/quick_start/clickHouse_example.md [hunk 13]
```diff
@@ -0,0 +1,93 @@
...
+![clickhouse](img/clickhouse/approval.png)
+
+返回 【数据集成】页面，等待任务配置成功
```
Comment (dockerzhang) [inline i18n/zh-CN/docusaurus-plugin-content-docs/current/quick_start/clickHouse_example.md:70]: ditto, and "返回 【数据集成】页面" -> 返回【数据集成】", more one space.

File: i18n/zh-CN/docusaurus-plugin-content-docs/current/quick_start/clickHouse_example.md [hunk 14]
```diff
@@ -0,0 +1,93 @@
...
+![clickhouse](img/clickhouse/result.png)
+
+## 测试数据
+### 发送数据
+![clickhouse](img/clickhouse/send_data.png)
+累计对 MySQL 添加 1001 条数据
+
+### 数据验证
+查看审计页面发送数据
```
Comment (dockerzhang) [inline i18n/zh-CN/docusaurus-plugin-content-docs/current/quick_start/clickHouse_example.md:79]: Add ":" at the end.

File: i18n/zh-CN/docusaurus-plugin-content-docs/current/quick_start/clickHouse_example.md [hunk 15]
```diff
@@ -0,0 +1,93 @@
...
+页面点击【审批管理】->【我的审批】->【详情】->【选择集群标签】->【通过】
+![clickhouse](img/clickhouse/approval.png)
+
+返回 【数据集成】页面，等待任务配置成功
+![clickhouse](img/clickhouse/result.png)
+
+## 测试数据
+### 发送数据
+![clickhouse](img/clickhouse/send_data.png)
+累计对 MySQL 添加 1001 条数据
```
Comment (dockerzhang) [inline i18n/zh-CN/docusaurus-plugin-content-docs/current/quick_start/clickHouse_example.md:76]: Add "。" at the end.

File: i18n/zh-CN/docusaurus-plugin-content-docs/current/quick_start/clickHouse_example.md [hunk 16]
```diff
@@ -0,0 +1,93 @@
...
+查看审计页面发送数据
+![clickhouse](img/clickhouse/data_page.png)
+
+然后进入 ClickHouse 容器，查看库表数据：
+![clickhouse](img/clickhouse/data_table.png)
+
+## 常见问题
+### 任务配置失败
+一般是 MQ 或者 Flink 集群配置错误导致，可以在页面查看错误信息，或者进入 Manager 容器查看详细日志。
+
```
Comment (dockerzhang) [inline i18n/zh-CN/docusaurus-plugin-content-docs/current/quick_start/clickHouse_example.md:88]: Too many empty lines.

File: i18n/zh-CN/docusaurus-plugin-content-docs/current/quick_start/clickHouse_example.md [hunk 17]
```diff
@@ -0,0 +1,93 @@
...
+
+
```
Comment (dockerzhang) [inline i18n/zh-CN/docusaurus-plugin-content-docs/current/quick_start/clickHouse_example.md:90]: Here lack the Q & A?

File: docs/quick_start/mysql_kafka_clickhouse_example.md [hunk 18]
```diff
@@ -0,0 +1,89 @@
+---
+title: MySQL -> Kafka -> ClickHouse Example
+sidebar_position: 2
```
Comment (dockerzhang) [inline docs/quick_start/mysql_kafka_clickhouse_example.md:3]:
```suggestion
sidebar_position: 5
```

File: i18n/zh-CN/docusaurus-plugin-content-docs/current/quick_start/mysql_kafka_clickhouse_example.md [hunk 19]
```diff
@@ -0,0 +1,86 @@
+---
+title: MySQL -> Kafka -> ClickHouse 示例
+sidebar_position: 2
```
Comment (dockerzhang) [inline i18n/zh-CN/docusaurus-plugin-content-docs/current/quick_start/mysql_kafka_clickhouse_example.md:3]:
```suggestion
sidebar_position: 5
```

**Review by dockerzhang — APPROVED** [review]

**Review by leezng — APPROVED** [review]

**Pull request merged**